service/src/test/java/bio/terra/workspace/service/privateresource/PrivateResourceCleanupServiceTest.java

```diff
@@ -204,20 +207,23 @@ void cleanupResourcesSuppressExceptions_cleansApplicationPrivateResource_succeed
         "grantWorkspaceRole");
     // Enable the WSM test app in this workspace. This has a test user as the "service account" so
     // we can delegate credentials normally.
-    App appConfig = getAppBySa(applicationAccessUtils.getApplicationSaEmail());
-    UUID appId = UUID.fromString(appConfig.getIdentifier());
+    WsmApplication app = applicationDao.getApplication(TEST_WSM_APP);
+    AccessToken saAccessToken = userAccessUtils.generateAccessToken(app.getServiceAccount());
```

Comment:
nit: usually I expect classes ending in "Utils" to be all static methods.

Reply:
Not my code. As I understand it, the `userAccessUtils` is an `@Component` so it can get some configuration.